content/references/http-websocket-apis/public-api-methods/clio-methods/nft_info.md

```diff
@@ -0,0 +1,135 @@
+---
+html: nft_info.html
+parent: clio-methods.html
+blurb: Retrieve information about the specified NFT using Clio server's `nft_info` API.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# nft_info
+[[Source]](https://github.com/XRPLF/clio/blob/4a5cb962b6971872d150777881801ce27ae9ed1a/src/rpc/handlers/NFTInfo.cpp "Source")
+
+The `nft_info` command asks the Clio server for information about the [NFT](non-fungible-tokens.html) being queried. 
```

Comment:
You could add a badge to mark when this feature was added and link the Clio release notes like this:

```suggestion
The `nft_info` command asks the Clio server for information about the [NFT](non-fungible-tokens.html) being queried. [New in: Clio v1.1.0](https://github.com/XRPLF/clio/releases/tag/1.1.0 "BADGE_BLUE")
```

(The version number and URL are my speculation. They don't currently exist, but if 1.1.0 is in fact the version the link should start working when the release is tagged on GitHub.)